neuronpp/utils/record.py

```diff
@@ -106,6 +108,12 @@ def _plot_animate(self, steps=10000, y_lim=None, position=None):
             * position=(3,3) -> if you have 9 neurons and want to display 'v' on 3x3 matrix
             * position='merge' -> it will display all figures on the same graph.
             * position=None -> Default, each neuron has separated  axis (row) on the figure.
+        :param true_class: list of true class labels in this window
+        :param pred_class: list of predicted class labels in window
+        :param stepsize: agent readout time step
+        :param dt: agent integration time step
+        :param show_true_predicted: whther to print true/predicted class' marks on the plot
```

Comment:
I think there is no point of using `show_true_predicted` param, because you can just check whether `true_class` and `pred_class` is not None.

Reply:
This is a matter of liking. Lots of code is written so that switch some behaviour on/off using a single switch. 
But this is an important comment: I shall move out the binary show_true_predicted out of the run_params tuple. Even so as this is not actually a simulation parameter.